src/components/RunMap/index.tsx

```diff
@@ -86,19 +83,27 @@ const RunMap = ({
     [endLon, endLat] = points[points.length - 1];
   }
   let dash = USE_DASH_LINE && !isSingleRun ? [2, 2] : [2, 0];
+  const onMove = React.useCallback(({ viewState }) => {
+    setViewState(viewState);
+  }, []);
+  const style: React.CSSProperties = {
+    width: '100%',
+    height: MAP_HEIGHT,
+    '& .mapboxgl-children': {
+      display: 'none',
+    },
+  };
 
   return (
-    <ReactMapGL
-      {...viewport}
-      width="100%"
-      height={MAP_HEIGHT}
-      mapStyle="mapbox://styles/mapbox/dark-v10"
-      onViewportChange={setViewport}
+    <Map
+      viewState={{ ...viewState }}
+      onMove={onMove}
+      style={style}
+      mapStyle="mapbox://styles/mapbox/dark-v9"
```

Comment:
Using `v10` or `v11` or zh language won't work, also use `{MAPBOX_TOKEN}` for the next 2 line
![image](https://github.com/yihong0618/running_page/assets/6956444/a24440be-4505-42c1-a814-e399651a3243)